tests/autoyast/autoyast_verify.pm

```diff
@@ -21,8 +21,10 @@ use testapi;
 sub run {
     my $self = shift;
     $self->result('fail'); # default result
-    
-    type_string "curl '" . get_var("AUTOYAST_VERIFY") . "' > verify.sh\n";
+
+    #todo: get the ip addresses by some function
+    my $verify_url .= 'http://10.0.2.1/data/' . get_var("AUTOYAST_VERIFY");
```

Comment:
or do you prefer to add some autoinst_url-like function in testapi.pm to return it?